data_structures/src/superblock.rs

```diff
@@ -171,6 +190,53 @@ impl SuperBlockState {
         }
     }
 
+    /// Return true if the local superblock has the majority of votes
+    pub fn has_consensus(&self) -> SuperBlockConsensus {
```

Comment:
should we add the line of the Tomasz's PR where he uses this function? To check if it is doing well. It is not necessary to do anything with the result of this function but al least call it